src/azure-cli/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -284,7 +284,7 @@ def load_arguments_eh(self, _):
                             'Once the isEnabled is set to false, all the existing connections of application group gets dropped and no new connections will be allowed')
 
     with self.argument_context('eventhubs namespace application-group create') as c:
-        c.argument('client_app_group_identifier', options_list=['--client-app-group-identifier', '--client-app-group-id'], help='The Unique identifier for application group.Supports SAS(SASKeyName=KeyName) or AAD(AADAppID=Guid)')
+        c.argument('client_app_group_identifier', options_list=['--client-app-group-identifier', '--client-app-group-id'], help='The Unique identifier for application group.Supports SAS(NamespaceSASKeyName=KeyName or EntitySASKeyName=KeyName) or AAD(AADAppID=Guid)')
```

Comment:
This would be a breaking change. Can we keep the `SASKeyName` until next breaking change window? During this period, we can 
- show warning to customers using `SASKeyName` to tell the deprecation info and
- automatically replace `SASKeyName` with `NamespaceSASKeyName`

What do you think?

Reply:
Since ClientAppGroupIdentifier is a string, customers can still provide for both of "SASKeyName=KeyName" and "NamespaceSASKeyName=KeyName". It is now upto the service if it will warn or throw an error. There is not much we can do from client side

Reply:
@evelyn-ys , the service for now would still accept SASKeyName for some time and customers can still provide for it, but it will be deprecated in the future hence making these doc changes to provide up to date info. Does this work for you ?